config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -191,6 +191,12 @@ presubmits:
       repo: cluster-api-provider-azure
       base_ref: release-1.9
       path_alias: sigs.k8s.io/cluster-api-provider-azure
+      workdir: true
```

Comment:
This should be false otherwise we won't find `./capz/run-capz-e2e.sh`. we want working dir to be the sig-windows test repo

Reply:
whoops

Reply:
updated - good catch!